crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -269,3 +269,73 @@ impl BaseClient {
         })
     }
 }
+
+#[cfg(test)]
+mod test {
+    use ruma::{device_id, room_id, user_id, RoomId};
+
+    use super::*;
+    use crate::SessionMeta;
+
+    #[tokio::test]
+    async fn can_process_empty_sliding_sync_response() {
+        let client = logged_in_client().await;
+        let empty_response = v4::Response::new("5".to_owned());
+        client.process_sliding_sync(&empty_response).await.expect("Failed to process sync");
+    }
+
+    #[tokio::test]
+    async fn process_sliding_sync_response_to_add_a_room() {
+        // Given a logged-in client
+        let client = logged_in_client().await;
+        let room_id = room_id!("!r:e.uk");
+
+        // When I send sliding sync response containing a room (with idenifiable data in
+        // joined_count)
+        let mut room = v4::SlidingSyncRoom::new();
+        room.joined_count = Some(41.try_into().expect("Failed to make UInt"));
```

Comment:
You can use `uint!(41)` or `41u32.into()`.

Reply:
Done in 4a4e143575